models/download-ggml-model.sh

```diff
@@ -22,7 +22,28 @@ function get_script_path() {
 models_path="$(get_script_path)"
 
 # Whisper models
-models=( "tiny.en" "tiny" "base.en" "base" "small.en" "small.en-tdrz" "small" "medium.en" "medium" "large-v1" "large" )
+models=( 
+    "tiny.en"
+    "tiny"
+    "model-whisper-tiny-q5_1"
+    "model-whisper-tiny.en-q5_1"
+    "base.en"
+    "base"
+    "model-whisper-base-q5_1"
+    "model-whisper-base.en-q5_1"
+    "small.en"
+    "small.en-tdrz"
+    "small"
+    "model-whisper-small-q5_1"
+    "model-whisper-small.en-q5_1"
+    "medium"
+    "medium.en"
+    "model-whisper-medium-q5_0"
+    "model-whisper-medium.en-q5_0"
+    "large-v1"
+    "large"
+    "model-whisper-large-q5_0"
+)
```

Comment:
Ah, I didn't notice that the quantum models are named `ggml-model-whisper-medium-q5_0.bin` instead of `ggml-medium-q5_0.bin`:

https://huggingface.co/ggerganov/whisper.cpp/tree/main

Are you interested in fixing the names both on HF and in this PR?

Reply:
https://huggingface.co/ggerganov/whisper.cpp/discussions/8/files

Reply:
It seems PR 8 just deleted the files instead of renaming them

Reply:
Yeah sorry, fixing now.

Reply:
@ggerganov good to go https://huggingface.co/ggerganov/whisper.cpp/discussions/9/files